packages/core/src/util/config.ts

```diff
@@ -453,4 +454,41 @@ export class Config {
   static getInternalServerUrl(): string {
     return getEnvVarOrFail("INTERNAL_SERVER_BASE_URL");
   }
+
+  static getAlohrIngestionBucket(): string {
+    return getEnvVarOrFail("ALOHR_INGESTION_BUCKET_NAME");
+  }
+
+  static getAlohrIngestionRemotePath(): string {
+    return getEnvVarOrFail("ALOHR_INGESTION_REMOTE_PATH");
+  }
+
+  static getAlohrIngestionPasswordArn(): string {
+    return getEnvVarOrFail("ALOHR_INGESTION_PASSWORD_ARN");
+  }
+
+  static getAlohrIngestionSftpConfig(): Partial<SftpConfig> {
+    const record = getEnvVarAsRecordOrFail("ALOHR_INGESTION_SFTP_CONFIG");
+
+    if (
+      !record.host ||
+      typeof record.host !== "string" ||
+      !record.port ||
+      typeof record.port !== "number" ||
+      !record.username ||
+      typeof record.username !== "string"
+    ) {
+      throw new Error("ALOHR_INGESTION_SFTP_CONFIG is not a valid SftpConfig");
+    }
+
+    return {
+      host: record.host,
+      port: record.port,
+      username: record.username,
+    };
```

Comment:
This is what zod is for, let's use it 😁 

On a related note, raf and i had a literal 30 - 45m debate over putting logic in these config getters. I 'disagree-and-commit'ed, so see how its done for the getHieConfigDictionary() - it just gets the value and then in a folder in domain we validate the structure of the object with zod.